### ISSUE TITLE
Multiple humanoid in Unity

### ISSUE THREAD

**Issue by TianhongZhou** [google-deepmind/mujoco#1347]
Hi, I currently use the plugin for Unity, and successfully import a mujoco humanoid. I also have a script (https://github.com/google-deepmind/mujoco/issues/1168#issuecomment-1795168565 from the help in this issue) to assign the qpos and qvel to this humanoid. I am wondering is there any way to have multiple humanoids in the same scene at the same time with different motions?

**Comment by Balint-H** [comment]
Yes, you can, just like in regular MuJoCo. It's as simple as copy-pasting the humanoid in the hierarchy window. Alternatively you can import a scene with multiple humanoids already. Similarly to regular MuJoCo you might have to take a bit of additional consideration to make sure you are affecting the correct DoFs when setting/reading state.  Can you tell a bit more about your use case?
